docs/source/traitsui_user_manual/view.rst

```diff
@@ -244,7 +244,7 @@ Visibility and status
    :term:`Handler` (see :ref:`controlling-the-interface-the-handler`).
 
    enabled_when: str
-      Expression that determines whether of group can be edited.
+      Expression that determines whether the group can be edited.
```

Comment:
It'll be very useful to add a note somewhere here - similar to the node that we added in code. It's always better to mention this in user facing docs instead of in code.

Reply:
and repeat the same comment a second time later in this file where we document `enabled_when`, `visible_when` again.

Reply:
I've added some comments here.  One thing I am noticing is that `defined_when` seems to be handled differently internally... I was going to make a general comment about all 3 at the bottom, but looking at `traitsui.ui.py` it appears `defined_when` is doing something slightly different?  I need to look at the code more closely.  For now I just added slightly redundant comments to both the enabled_when and visible_when description